filebeat-yml-script.py

```diff
@@ -73,7 +73,7 @@ def _is_open():
 
 def _add_shipping_data():
     yaml = YAML()
-    with open("docker-colletor-logs/default_filebeat.yml") as default_filebeat_yml:
+    with open("default_filebeat.yml") as default_filebeat_yml:
```

Comment:
Can you please double check? I think it won't work without the `docker-collector-logs/` prefix